klayout_package/python/kqcircuits/simulations/simulation.py

```diff
@@ -1455,3 +1455,39 @@ def visualise_region(
         else:
             for i, p in enumerate(points):
                 self.cell.shapes(visualisation_layer).insert(pya.DText(f"{label}_{i+1}", p.x, p.y))
+   def visualise_ports(self):
```

Comment:
```suggestion
   def visualise_ports(self, thickness=50):
```
The port visualisation thickness could be passed as argument, but given some default value like 50 microns. Then this cab decide the thickness of the shape for both internalport and edgeport